tools/concat_release_notes.py

```diff
@@ -0,0 +1,47 @@
+from packaging import version
```

Comment:
This script comes from qiskit-ibm-runtime. Let me know if you prefer it to be in a different folder like `scripts/`.

Reply:
For me is more descriptive `scripts/`, but let see what @jesus-talavera-ibm and @cbjuan think :)

Reply:
Yes, I also think that `scripts/` is more understandable